### ISSUE TITLE
Mt. Pelerin KYC and AML integration

### ISSUE THREAD

**Issue by BvL13** [commons-stack/website#3]
The Mt. Pelerin plugin will be integrated into a special section of the commons stack website that is not directly available through the menu.
The link to this part of the website will be communicated by the commons stack association board to every user that successfully applied for membership.

This plugin takes the user through the KYC & AML process defined together with Mt. Pelerin. This includes the T&C of Mt. Pelerin.

**Comment by MaxSemenchuk** [comment]
What should i do now? remove the form from the homepage? create a new page?

**Comment by BvL13** [comment]
Please create a new page for each credential that we receive from Mt. Pelerin.
The credentials will be covering a certain amount of payment by the user. This means you will have to create three different URLs and we send the correct URL to the person every time we know the amount he will invest.

**Comment by MaxSemenchuk** [comment]
Waiting for credentials and then can check how to do it better

**Comment by MaxSemenchuk** [comment]
https://commonsstack.org/kyc1
https://commonsstack.org/kyc2
https://commonsstack.org/kyc3

@BvL13 pls check

**Comment by GriffGreen** [comment]
I can't get my birthday to work
<img width="521" alt="Screen Shot 2019-06-14 at 2 49 38 PM" src="https://user-images.githubusercontent.com/14945613/59510327-b0eb1f80-8eb3-11e9-9470-3eee3a85ea5f.png">

**Comment by GriffGreen** [comment]
this was on kyc2... also there isnt a clear label on which one

**Comment by GriffGreen** [comment]
ok figured it out but its a really bad UX if you try to type it in...

**Comment by BvL13** [comment]
@MaxSemenchuk I dont see oiur T&C included in here. Where does the user have to klick on the Commons Stack Association Terms and Conditions?

**Comment by BvL13** [comment]
I will also ask Mt. Pelerin to remove the token sale part. Its a membership they receive, no tokens involved.
The Ethereum address also has to be communicated differently. I will come up with a wording for that.

**Comment by MaxSemenchuk** [comment]
@BvL13 it used to be in the footer on all website pages, but @krrisis removed it. Do you mean it should be in the footer only on KYC pages?

**Comment by krrisis** [comment]
yeah, it needs to be on the KYC page(s), but not on the homepage

**Comment by BvL13** [comment]
Correct. Only on the KYC Page where you also sign the T&C of Mt. Pelerin

**Comment by BvL13** [comment]
@MaxSemenchuk we changed the process with Mt.Pelerin. We now only have one link:
https://www.mtpelerin.com/customers/commons-stack.html?code=kxDHQeRGz3HxfXT5eUhCaZbdBkHKQt9Kzi3otVtY

It asks you the amount first and then leads you to the proper KYC for the amount you are willing to pay. 
Sorry for asking you again to change the KYC I-Frame link to the one above.
Please also ensure that the T&C from our side are integrated.
